public/index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Groceries List</title>
+  </head>
+
+  <body>
+    <h1>Groceries List</h1>
+    <hr />
+    <p id="message"></p>
+    <hr />
+    <div id="logon-register" style="display:none;">
+      <button type="button" id="logon">logon</button>
+      <button type="button" id="register">register</button>
+    </div>
+    <div id="logon-div" style="display:none;">
+      <form>
+        <div>
+          <label for="email">email:</label>
+          <input type="email" id="email" />
+        </div>
+        <div>
+          <label for="password">password:</label>
+          <input type="password" id="password" />
+        </div>
+        <button type="button" id="logon-button">logon</button>
+        <button type="button" id="logon-cancel">cancel</button>
+      </form>
+    </div>
+    <div id="register-div" style="display:none;">
+      <form>
+        <div>
+          <label for="name">name:</label>
+          <input type="text" id="name" />
+        </div>
+        <div>
+          <label for="email1">email:</label>
+          <input type="email" id="email1" />
+        </div>
+        <div>
+          <label for="password1">password:</label>
+          <input type="password" id="password1" />
+        </div>
+        <div>
+          <label for="password2">verify password:</label>
+          <input type="password" id="password2" />
+        </div>
+        <button type="button" id="register-button">register</button>
+        <button type="button" id="register-cancel">cancel</button>
+      </form>
+    </div>
+    <div id="groceries" style="display:none;">
+      <p id="groceries-message"></p>
+      <table id="groceries-table">
+        <tr id="groceries-table-header">
+          <th>Name</th>
+          <th>Added By</th>
+          <th>Status</th>
+          <th colspan="2"></th>
+        </tr>
+      </table>
+      <button type="button" id="add-product">add product</button>
+      <button type="button" id="logoff">log off</button>
+    </div>
+    <div id="edit-groceries" style="display:none;">
+      <form>
+        <div>
+          <label for="name">name:</label>
+          <input type="text" id="productName" />
+        </div>
+        <div>
+          <label for="status">status:</label>
+          <select id="status">
+            <option value="pending">pending</option>
+            <option value="bought">bought</option>
+            <option value="notfound">not found</option>
```

Comment:
Change `notfound` for `not found`, because in you model it expected to receive `not found`.